src/Corporation/CorporationState.ts

```diff
@@ -9,20 +9,21 @@ export class CorporationState {
   // Get the name of the current state
   // NOTE: This does NOT return the number stored in the 'state' property,
   // which is just an index for the array of all possible Corporation States.
-  getState(): CorpStateName {
+  get nextState(): CorpStateName {
```

Comment:
maybe call these `state.nextName` and `state.prevName`?

Reply:
what do you think about 
.nextStateName
.prevStateName ?

Reply:
oh nvm the state is already in there yeah .nextName .prevName sound good